<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="first_x_coord_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_y_coord_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="second_y_coord_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="second_x_coord_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="start_point">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Начальная точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_point">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Средняя точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_coord">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_coord">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Нарисовать</string>
    </property>
   </widget>
   <widget class="QSlider" name="rotateSlider">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="end_point_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол поворота</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_point_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="third_y_coord_edit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="third_x_coord_edit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="end_point_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Конечная точка</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
